flexget/utils/qualities.py

```diff
@@ -253,18 +260,24 @@ def _find_best(self, qlist, default=None, strip_all=True):
 
     @property
     def name(self):
-        name = ' '.join(str(p) for p in (self.resolution, self.source, self.codec, self.audio) if p.value != 0)
+        name = ' '.join(str(p) for p in (self.resolution, self.source, self.codec, self.audio) +
+                        tuple(self.custom_components) if p.value != 0)
         return name or 'unknown'
 
     @property
     def components(self):
-        return [self.resolution, self.source, self.codec, self.audio]
+        # TODO lexicographical ordering maybe doesn't make sense anymore?
+        return self.custom_components + [self.resolution, self.source, self.codec, self.audio]
```

Comment:
I'm a little unsure about this one. Apparently we use lexicographical ordering when comparing two qualities. That means we must put the custom components somewhere that makes sense, which begs the question; do we still want lexicographical ordering?

Reply:
lexicographical ordering? All of the components define their own ordering based on their numeric value. Not sure what you mean here.

Reply:
I don't remember 100%, but iirc the comparison between two qualities compares the qualities in order, so if the resolution is not equal to or greater than the other, then it's a "worse" quality.

Reply:
Yeah, it does do that. Came up in chat as well, where should the custom component go in that order, and does it need to be configurable. Not sure I have any good answers.